morphia/src/main/java/org/mongodb/morphia/mapping/Mapper.java

```diff
@@ -737,7 +755,7 @@ private void addConverters(final MappedClass mc) {
     private MappedClass addMappedClass(final MappedClass mc, final boolean validate) {
         addConverters(mc);
 
-        if (validate) {
+        if (validate && !mc.isInterface()) {
```

Comment:
If I remove this change all the tests still pass.

I suggest reverting this and handling in a separate Issue/PR.